vortex-array/src/view.rs

```diff
@@ -156,11 +262,36 @@ impl<'v> IntoArray<'v> for ArrayView<'v> {
 #[derive(Debug)]
 pub struct ViewContext {
     encodings: Vec<EncodingRef>,
+    stats: Vec<Stat>,
 }
 
 impl ViewContext {
-    pub fn new(encodings: Vec<EncodingRef>) -> Self {
-        Self { encodings }
+    pub fn new(encodings: Vec<EncodingRef>, stats: Vec<Stat>) -> Self {
+        Self { encodings, stats }
+    }
+
+    pub fn set_stats(&mut self, to_enable: &[Stat]) {
```

Comment:
Added this set_stats method so that we can default to including stats but allow a caller to smash over this with any combination of stats they want. 

The default mechanism is not implemented because there is an open question as to what the right set of default encodings is. I think using default stats in the from() method below is reasonable behavior, given that it can be modified afterwards.